### DISCUSSION TITLE
Intragranular misorientation distribution of selected grain

### DISCUSSION THREAD

**Discussion by Mustafa-Rifat** [mtex-toolbox/mtex#555] [Ask Anything] [answered]
Hello MTEX Community,

I want to find intragranular misorientation of selected grains from my ebsd plot. I am trying to do something like this demo page: https://mtex-toolbox.github.io/HomepageOld/files/doc/GrainMisorientationAnalysis.html#22

I got some help from this page also: https://mtex-toolbox.github.io/GrainOrientationParameters.html

Now what I am trying is like this:
```matlab
[grains,ebsd.grainId] = calcGrains(ebsd);
%My grain Ids are 489 and 490
ori = ebsd(ebsd.grainId == 490).orientations;
mis2mean = inv(grains(490).meanOrientation) .* ori;
figure
plotAngleDistribution(mis2mean)
```
This gives me a plot like this:
plot.jpg
If I do the same for grain 489, I get another plot like this.
However, I want a single histogram containing data for both grains. I want to tell by looking at the histogram how the distribution looks like together i.e. is the distribution bimodal (the grains have very different intragranular misorientation distribution) or not.

Can anyone help me in this regard?
Thank you!

**Comment by ralfHielscher** [comment]
Do you want to have plotted the one distribution on top of the other distribution or do you want to have the common distribution of both grains?

Ralf.

**Comment by Mustafa-Rifat** [comment]
Common distribution of both grains i.e. one histogram showing
misorientation distribution combinedly.

**Comment by zmichels** [comment, marked as answer]
Why not just combine the results for the separate grains and plot them?

oris489 = ebsd(grains(489)).orientations;
mis2mean489 = inv(grains(489).meanOrientation) .* oris489;

oris490 = ebsd(grains(490)).orientations; 
mis2mean490 = inv(grains(490).meanOrientation) .* oris490;

mis2meanGrains = [mis2mean489; mis2mean490];

figure
plotAngleDistribution(mis2meanGrains)

**Reply by Mustafa-Rifat** [reply]
Thanks! This is what I wanted.

**Comment by zmichels** [comment]
If you truly want the mis2mean instead of neighbor-to-neighbor misorientations, you can shorten the example I sent a minute into one line of code as this:

plotAngleDistribution(ebsd(grains(489:490)).mis2mean)


Hope this helps.

Cheers,
Zach
